pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -52,18 +53,22 @@ func validateVanillaControllerUnpublishVolumeRequest(ctx context.Context, req *c
 // validateVanillaControllerExpandVolumeRequest is the helper function to validate
 // ExpandVolumeRequest for Vanilla CSI driver.
 // Function returns error if validation fails otherwise returns nil.
-func validateVanillaControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest) error {
+func validateVanillaControllerExpandVolumeRequest(ctx context.Context, req *csi.ControllerExpandVolumeRequest,
+	containerOrchestratorUtility commonco.COCommonInterface) error {
```

Comment:
Can you pass the feature state (bool) as a function parameter rather than the entire CO interface object?

Reply:
Done